### DISCUSSION TITLE
Star an issue

### DISCUSSION THREAD

**Discussion by jenyang2** [community/community#35987] [Projects and Issues]
I need the ability to star an issue.

**Comment by Aboraneem1400** [comment]
في الخميس، ١٣ أكتوبر، ٢٠٢٢ ٨:٤٠ ص jenyang2 ***@***.***> كتب:

**Reply by jenyang2** [reply]
??

**Comment by queenofcorgis** [comment]
Hi @jenyang2 - thanks for posting! Mind elaborating on how or why this would be helpful for you to give our team greater context 😄.

**Comment by spenserblack** [comment]
Was about to make a duplicate issue until I stumbled on this :laughing:

For me, I'd like to be able to star issues because sometimes I stumble on an issue that I might want to work on later. I'm not a maintainer so I can't self-assign. Additionally, I haven't decided if I want to take on the issue yet, so I don't want to ask maintainers to assign it to me. Starring an issue would be a way for me to put the issue in a list of potential contributions that I haven't committed to yet.

Another use case I can think of is that sometimes there are issues that I want to check out every once in a while to check on the discussion, but I don't want to subscribe (perhaps it's very active and generates too many notifications).

**Comment by ntmt2903** [comment]
In my case, I want to star issues because sometimes the solution I was looking for is in the discussion. It takes time to find solutions to issues interesting only for people in a niche research community. It saves time when I need to refer the solution to somebody, for example my colleagues, who might run into the same problem.
